corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -142,9 +142,11 @@ public int printTable(String namespace, String tablename) {
                     builder = new StringBuilder("\nKey:\n")
                             .append(JsonFormat.printer().print(entry.getKey().getKey()))
                             .append("\nPayload:\n")
-                            .append(JsonFormat.printer().print(entry.getValue().getPayload()))
+                            .append(entry.getValue() != null && entry.getValue().getPayload() != null ?
```

Comment:
Is it possible that value is null?

Reply:
I checked the code and it is not possible that value is null.  But this is how it has been changed in Impactor(check for value == null) so I have reused the same approach.